.gitignore

```diff
@@ -37,3 +37,4 @@ kontainer-engine
 k3s-images.txt
 k3s-airgap-images.tar
 data.json
+*~
```

Comment:
Do we need this?

Reply:
I use emacs sometimes for quick edits and it leaves those behind as quick backups. I'd like to turn this off for files maintained in `git` -- seems like a one-day hackweek project, as there's a fair bit of work to do in elisp.